src/protocol/logpoint.ts

```diff
@@ -93,14 +98,20 @@ async function showPrimitiveLogpoints(
   }
 }
 
-function saveLogpointHits(points: PointDescription[], locations: Location[]) {
-  if (PointHandlers.onPoints) {
-    PointHandlers.onPoints(points, { locations });
+function saveLogpointHits(
+  points: PointDescription[],
+  results: AnalysisEntry[],
+  locations: Location[],
+  condition: string
+) {
+  if (condition) {
+    points = points.filter(point =>
+      results.some(result => result.key === point.point && result.value.time === point.time)
```

Comment:
what is this doing?

Reply:
Filtering out the `PointDescription`s referring to points where the condition is false.